### PR TITLE
Peer code review.

### PR DESCRIPTION
**Pull request by carlylechia**
This branch was created for a code review from my coding partner. Can't wait to get his feedback.

### CODE REVIEW

**Comment by bushmusi** [comment]
## Hi @carlylechia  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

**Review by bushmusi — APPROVED** [review]
## Hi @carlylechia  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

**Comment by carlylechia** [comment]
😅🤩 You're going to make an AMAZING code reviewer some day man. Thank you so much for your beautiful reviews, and for the approval.

File: modules/getStates.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const setState = (tasks, checkbox, index) => {
+  if (checkbox.checked === true) {
+    tasks[index].checked = true;
+  } else {
+    tasks[index].checked = false;
+  }
```
Comment (bushmusi) [inline modules/getStates.js:6]: You have done a great job populating checkbox status 🥇 . What I kindly suggest you is to use `inline if conditions`

**Possible suggestion**
`tasks[index].checked = checkbox.checked === true ? true : false` 

This way your code will look `CLEAN`

File: index.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  taskWrapper.classList.add('drag-list');
+  const mylocal = getFromLocalStorage();
+  mylocal.forEach((tsk, index) => {
+    const li = document.createElement('li');
+    li.setAttribute('draggable', 'true');
+    li.setAttribute('data-index', index);
+    li.classList.add('draggable');
+    const checkbox = document.createElement('input');
+    checkbox.setAttribute('type', 'checkbox');
+    // eslint-disable-next-line no-unused-expressions
```
Comment (bushmusi) [inline index.js:41]: I suggest you remove the `eslint-disable` command. Fixing errors will help you to follow **JS best practice** and you will get professional coding skill through fixing them

File: index.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    tsk.checked ? checkbox.setAttribute('checked', true) : null;
+
+
+    checkbox.addEventListener('change', (e) => {
+      e.preventDefault();
+      // eslint-disable-next-line no-use-before-define
```
Comment (bushmusi) [inline index.js:47]: It's same here as I said above

**Pull request merged**